.rubocop_todo.yml

```diff
@@ -17,18 +17,26 @@ Layout/EmptyLineAfterGuardClause:
   Exclude:
     - 'lib/common/models/collection.rb'
 
-# Offense count: 7
+# Offense count: 6
 # Cop supports --auto-correct.
+# Configuration parameters: AllowAliasSyntax, AllowedMethods.
+# AllowedMethods: alias_method, public, protected, private
 Layout/EmptyLinesAroundAttributeAccessor:
   Exclude:
-    - 'lib/common/models/collection.rb'
```

Comment:
Did you mean to remove this file?

Reply:
Yep -- that was fixed at some point and rubocop just picked it up :)

Reply:
Oh cool 😎